service/src/main/java/bio/terra/axonserver/service/cloud/gcp/CloudStorageService.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.axonserver.service.cloud.gcp;
+
+import bio.terra.axonserver.service.exception.CloudObjectReadException;
+import bio.terra.axonserver.utils.BoundedByteArrayOutputStream;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.ReadChannel;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.StorageOptions;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+/** Service for interacting with Google Cloud Storage */
+public class CloudStorageService {
```

Comment:
Spring naming-wise, this isn't a `Service` (the thing you'd annotate with `@Service`), since it's not injected and has no state. It looks like a `utils` type class to me.